vllm/worker/model_runner.py

```diff
@@ -49,8 +49,8 @@
 from vllm.sequence import IntermediateTensors, SequenceGroupMetadata
 from vllm.transformers_utils.config import uses_mrope
 from vllm.utils import (DeviceMemoryProfiler, PyObjectCache, async_tensor_h2d,
-                        flatten_2d_lists, is_pin_memory_available,
-                        supports_dynamo, weak_ref_tensor)
+                        flatten_2d_lists, is_hip, is_pin_memory_available,
```

Comment:
why `is_hip` ?

Reply:
oops, it seems I should remove this one. Thanks!